src/sentry/api/endpoints/organization_details.py

```diff
@@ -186,7 +186,7 @@ def save(self):
                 # default value evaluates as truthy, but this should work for now with the
                 # current ORG_OPTIONS (assumes ORG_OPTIONS are falsy)
                 if type_(self.init_data[key]):
-                    changed_data[key] = self.init_data[key]
+                    changed_data[key] = u'to {}'.format(self.init_data[key])
```

Comment:
I don't know how difficult it is to do, but I find that logs are most helpful when they have both before and after. From one value to another. That said if it's a ton of work, I wouldn't bother; this is much, much better than logging nothing at all.

Reply:
This is only when there's no previous value. Otherwise the format is `from x to y`.